macroslib/tests/expectations/foreign_enum_plus_interface.cpp

```diff
@@ -1,7 +1,9 @@
 "virtual void onSessionUpdate(ControlItem item, bool is_ok) noexcept = 0;";
 
 r#"enum ControlItem {
+
```

Comment:
It would be nice to not add unnecessary new lines,
because of somebody can read generated code.

So may be: 

```
if !doc_comments.is_empty() && !doc_comments.ends_with('\n') {
    doc_comments.push('\n');
}
```
?